### DISCUSSION TITLE
Icon request: icon-swipe, icon-pinch

### DISCUSSION THREAD

**Discussion by mediastuttgart** [FortAwesome/Font-Awesome#21170] [Icon Requests]
Would be great if some gesture icons like swipe, rotate, pinch, etc. were added :)

cheers
michael

**Comment by davegandy** [comment]
Rotate should be covered by icon-repeat and icon-undo. Have examples for the rest?

**Comment by bganotoo** [comment]
I like the general idea in the accepted answer at http://ux.stackexchange.com/questions/12504/symbol-for-swipe-left. The circle + arrow is common in user manuals for touch devices, and is fairly compact.

**Comment by tarwin** [comment]
I've found the best icons for gestures (the ones that you don't have to explain with text as well for general users to understand) are these - http://www.webiconset.com/gesture-vector-icon-set/.

They are proprietary I know, but it would be pretty amazing if the community could convince these guys to add them to FontAwesome, maybe through a crowdfunded donation.

It really would be good to have a standard go-to for gestures.

Thoughts from anyone?

**Comment by Nagadir** [comment]
+1 
I agree! Font Awesome needs more icons for touch based devices :)

Something like this (with hands and symbols) could do:

http://www.prosoxi.com/wp-content/uploads/2012/06/thumb.jpeg

**Comment by ryan-teliczan** [comment]
How do you still not have swipe icons? It's been nearly 7 years since the original request! Something must be wrong with the way you prioritize requests.

**Comment by bjornbjorn** [comment]
I was also looking for a "pinch"-icon :-)

**Comment by username1290** [comment]
Gestures!

**Comment by jnorkus** [comment]
9 years after, more relevant than ever before.

**Comment by tucomel** [comment]
9 years later and it still doesn't exist?! maybe it never will! sadtimes

**Comment by tucomel** [comment]
if someone want for free, I made 2 versions of swipe_up and swipe_down until fontawesome release they version
[swipe_down_up_regular_and_solid_svgs.zip](https://github.com/FortAwesome/Font-Awesome/files/6446855/swipe_down_up_regular_solid.zip)

**Comment by adashmada** [comment]
12 years on and still waiting... :/

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Reply by tucomel** [reply]
@adashmada your time has come
